core/api/src/app/wallets/get-invoices-for-wallets.ts

```diff
@@ -1,17 +1,22 @@
 import { MAX_PAGINATION_PAGE_SIZE } from "@/config"
-import { parsePaginationArgs } from "@/domain/pagination"
+import { checkedToPaginatedQueryArgs } from "@/domain/primitives"
 import { WalletInvoicesRepository } from "@/services/mongoose"
 
 export const getInvoicesForWallets = async ({
   wallets,
   paginationArgs,
 }: {
   wallets: Wallet[]
-  paginationArgs?: PaginationArgs
-}): Promise<PaginatedResult<WalletInvoice> | ApplicationError> => {
+  paginationArgs: {
+    first?: number | null
+    last?: number | null
+    before?: string | null
+    after?: string | null
+  }
+}): Promise<PaginatedQueryResult<WalletInvoice> | ApplicationError> => {
   const walletIds = wallets.map((wallet) => wallet.id)
 
-  const parsedPaginationArgs = parsePaginationArgs({
+  const parsedPaginationArgs = checkedToPaginatedQueryArgs({
```

Comment:
I guess the variable should now be `checkedPaginationArgs`? Or even better `paginationArgs` and rename the above one to `rawPaginationArgs`... the checked type should be the default one (ie simpler name) the _unchecked_ one should have the special name (ie prefix).

Reply:
Will make this change.